models/src/anemoi/models/models/encoder_processor_decoder.py

```diff
@@ -76,7 +76,17 @@ def __init__(
         self.multi_step = model_config.training.multistep_input
         self.num_channels = model_config.model.num_channels
 
-        self.node_attributes = NamedNodesAttributes(model_config.model.trainable_parameters.hidden, self._graph_data)
+        precision_mapping = {
```

Comment:
Can we describe the torch object in the hydra config directly? save creating this dict?